stock_manual_transfer/tests/test_stock_manual_transfer.py

```diff
@@ -8,6 +8,7 @@ class TestStock(TransactionCase):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
+        cls.env.user.groups_id += cls.env.ref("uom.group_uom")
```

Comment:
@luisg123v this shouldn't be done, there is a group restriction in the stock manual transfer view over the unit of measure in [here](https://github.com/Vauxoo/addons-vauxoo/blob/16.0/stock_manual_transfer/views/stock_manual_transfer_views.xml#L47) and in v15 it worked fine, the field didn't show and a user without the multi unit of measure group and they can make the manual transfers but now in the v16 you cannot preform a manual transfer if you don't belong to the multi unit group.

![image](https://github.com/Vauxoo/addons-vauxoo/assets/119946425/36d885ab-b64b-42d5-a3e0-e6086125e166)

cc: @allanpa88